examples/vnext/src/mastra/agents/test.ts

```diff
@@ -1,27 +1,29 @@
-import { Agent } from "@mastra/core";
-import { integrations } from "../integrations";
-import * as tools from "../tools";
+import { Agent } from '@mastra/core';
+
+import { integrations } from '../integrations';
+import * as tools from '../tools';
 
 export const agentOne = new Agent<typeof tools, typeof integrations>({
-    name: 'Agent One',
-    instructions: 'Do this',
-    model: {
-        provider: "GROQ_VERCEL",
-        name: "llama3-groq-70b-8192-tool-use-preview",
-        toolChoice: "required",
-    },
-    tools: {
-        testTool: true,
-        gmailGetProfile: true,
-    }
-})
+  name: 'Agent One',
+  instructions: 'Do this',
+  model: {
+    provider: 'GROQ_VERCEL',
+    name: 'llama3-groq-70b-8192-tool-use-preview',
+    toolChoice: 'required',
+  },
+  tools: {
```

Comment:
Didn't pull latest?
This is now `enabledTools`. `tools` represents the concrete tools collection.

Reply:
updated